docroot/profiles/custom/cgov_site/themes/custom/cgov/cgov_common/templates/content/home_landing/paragraph--cgov-card-internal.html.twig

```diff
@@ -32,5 +41,5 @@
   {% set image = content.field_featured_item %}
 
   {## Render Card ##}
-  {{cardMacros.featureCard(title, description, image, url)}}
+  {{ attribute(cardMacros, cardType, [title, description, image, url])}}
```

Comment:
Yeah, this DEFINITELY needs a comment. That is not a normal way to use attribute in any Drupal examples and caused some head scratching. (Neat usage though...)

Reply:
Switched back to an `if/else` statement using cardType variable to pick card macro.